rocAL/rocAL_pybind/examples/PYTHON_UNITTEST_TEST_FILE.sh

```diff
@@ -1,149 +1,118 @@
-            ? ) helpFunction ;; # Print helpFunction in case parameter is non-existent
-        esac
-    done
-
-    # Print helpFunction in case parameters are empty
+if [[ $ROCAL_DATA_PATH == "" ]]
+then 
+    echo "Need to export ROCAL_DATA_PATH"
+    exit
+fi
 
-    if [ -z "$backend" ];
-    then
-        backend_arg=no-rocal-gpu
-    else
-        if [[ $backend == "cpu" || $backend == "CPU" ]]; then
-            backend_arg=no-rocal-gpu
-        elif [[ $backend == "gpu" || $backend == "GPU" ]]; then
-            backend_arg=rocal-gpu
-        fi
+# Path to inputs and outputs available in MIVisionX-data
+image_path=${ROCAL_DATA_PATH}/rocal_data/coco/coco_10_img/train_10images_2017/
+coco_detection_path=${ROCAL_DATA_PATH}/rocal_data/coco/coco_10_img/train_10images_2017/
+tf_classification_path=${ROCAL_DATA_PATH}/rocal_data/tf/classification/
+tf_detection_path=${ROCAL_DATA_PATH}/rocal_data/tf/detection/
+caffe_classification_path=${ROCAL_DATA_PATH}/rocal_data/caffe/classification/
+caffe_detection_path=${ROCAL_DATA_PATH}/rocal_data/caffe/detection/
+caffe2_classification_path=${ROCAL_DATA_PATH}/rocal_data/caffe2/classification/
+caffe2_detection_path=${ROCAL_DATA_PATH}/rocal_data/caffe2/detection/
+mxnet_path=${ROCAL_DATA_PATH}/rocal_data/mxnet/
+output_path=./rocal_python_unittest_output_folder_$(date +%Y-%m-%d_%H-%M-%S)/
+golden_output_path=${ROCAL_DATA_PATH}/rocal_data/GoldenOutputsTensor/
+
+display=0
+batch_size=2
+device=0
+width=640 
+height=480
+device_name="host"
+rgb_name=("gray" "rgb")
+rgb=1
+dev_start=0
+dev_end=1
+rgb_start=0
+rgb_end=1
+
+if [ "$#" -gt 0 ]; then 
+    if [ "$1" -eq 0 ]; then # For only HOST backend
+        dev_start=0
+        dev_end=0
+    elif [ "$1" -eq 1 ]; then # For only HIP backend
+        dev_start=1
+        dev_end=1
+    elif [ "$1" -eq 2 ]; then # For both HOST and HIP backend
+        dev_start=0
+        dev_end=1
     fi
+fi
 
-    if [ -z "$number_of_gpus" ];
-    then
-        gpus_per_node=1
-    else
-        gpus_per_node=$number_of_gpus
+if [ "$#" -gt 1 ]; then
+    if [ "$2" -eq 0 ]; then # For only Greyscale inputs
+        rgb_start=0
+        rgb_end=0
+    elif [ "$2" -eq 1 ]; then # For only RGB inputs
+        rgb_start=1
+        rgb_end=1
+    elif [ "$2" -eq 2 ]; then # For both RGB and Greyscale inputs
+        rgb_start=0
+        rgb_end=1
     fi
+fi
 
+mkdir "$output_path"
 
-    if [ -z "$dump_outputs" ];
-    then
-        display_arg=display #True by default
-    else
-        if [[ $dump_outputs == "true" || $dump_outputs == "True" ]]; then
-            display_arg=display
-        elif [[ $dump_outputs == "false" || $dump_outputs == "False" ]]; then
-            display_arg=no-display
-        fi
-    fi
-
-        if [ -z "$print_tensor" ];
-    then
-        print_tensor_arg=print_tensor #True by default
+for ((device=dev_start;device<=dev_end;device++))
+do 
+    if [ $device -eq 1 ]
+    then 
+        device_name="hip"
+        backend_arg=rocal-gpu
```

Comment:
@fiona-gladwin Has this been tested on both docker and bare metal for both backends. Please talk to Swetha, I had a call with her regarding this

Reply:
I have tested the python unit tests on both docker and bare metal - its working as expected

Reply:
Lakshmi, Please check if the issue with the script still exist or not